cloudbuild.yaml

```diff
@@ -1,16 +1,16 @@
 steps:
         - name: 'ubuntu'
-          args: ['bash', 'conformance/start.sh', '$COMMIT_SHA']
+          args: ['bash', 'conformance/start.sh', '$COMMIT_SHA'] # writes start time and #PR to started.json
```

Comment:
This is a personal nit, but would you mind moving the end of line comment above the `args` in each case. I find it easier to follow the flow that way.